assets/app/scripts/controllers/newfromtemplate.js

```diff
@@ -131,11 +131,15 @@ angular.module('openshiftConsole')
           Navigate.toProjectOverview($scope.projectName);
         },
         function(result) { // failure
+          var details;
+          if (result.data && result.data.message) {
```

Comment:
Status is the http status code, so I would say we do, but thinking about it, we probably don't need to display that here. If there are details, then we should show them and that's it.